### PR TITLE
Thunderbolt 3 works with MacBookPro13,3

### PR DESCRIPTION
**Pull request by schwabe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -249,9 +249,12 @@ An accelerometer doesn't seem to be available at all.
...
+![MacBookPro13,1 unknown](https://img.shields.io/badge/MacBookPro13%2C1-unknown-lightgrey.svg) ![MacBookPro13,2 unknown](https://img.shields.io/badge/MacBookPro13%2C2-unknown-lightgrey.svg) ![MacBookPro13,3 working](https://img.shields.io/badge/MacBookPro13%2C3-working-green.svg) ![MacBookPro14,1 unknown](https://img.shields.io/badge/MacBookPro14%2C1-unknown-lightgrey.svg) ![MacBookPro14,2 unknown](https://img.shields.io/badge/MacBookPro14%2C2-unknown-lightgrey.svg) ![MacBookPro14,3 unknown](https://img.shields.io/badge/MacBookPro14%2C3-unknown-lightgrey.svg)
 
-Untested, but probably working with the following patch:
+Tested on the Macbook13,3 with a HP Thunderbolt 3 Dock. Displayport
+Port with a 4k Display with 60 Hz works.
+
+Untested on other models, but probably working with the following patch:
 https://gist.github.com/l1k/3ffbd63ac337fa85530f6db3e9ee77e9
```
Comment (roadrunner2) [inline README.md]: These PCI id's (and associated functionality) were added in 4.13, so the patch is definitely not necessary as of 4.13.

**Comment by Dunedan** [comment]
Thanks a lot for your contribution. :+1:
Great to get confirmation that Thunderbolt works.

Aside from what @roadrunner2 already pointed out I think it's fair to assume that all MacBook Pro models since 2016 work with Thunderbolt, if one of them works, as they all use the same `Intel Corporation DSL6540 Thunderbolt 3 Bridge [Alpine Ridge 4C 2015] [8086:1578] (rev 02)` chips. Can you please adjust your PR and mark all models as supported?!

**Comment by schwabe** [comment]
@Dunedan I updated the PR. A slight side note: The Macbooks use Alpine Ridge 4C 2016, the Alpine Ridge 4C 2015 in the lspci is the chip in the dock.

File: README.md [hunk 2]
```diff
@@ -249,9 +249,14 @@ An accelerometer doesn't seem to be available at all.
 
 ## Thunderbolt
 
-![MacBookPro13,1 unknown](https://img.shields.io/badge/MacBookPro13%2C1-unknown-lightgrey.svg) ![MacBookPro13,2 unknown](https://img.shields.io/badge/MacBookPro13%2C2-unknown-lightgrey.svg) ![MacBookPro13,3 unknown](https://img.shields.io/badge/MacBookPro13%2C3-unknown-lightgrey.svg) ![MacBookPro14,1 unknown](https://img.shields.io/badge/MacBookPro14%2C1-unknown-lightgrey.svg) ![MacBookPro14,2 unknown](https://img.shields.io/badge/MacBookPro14%2C2-unknown-lightgrey.svg) ![MacBookPro14,3 unknown](https://img.shields.io/badge/MacBookPro14%2C3-unknown-lightgrey.svg)
+![all models working](https://img.shields.io/badge/all_models-working-green.svg)
```
Comment (Dunedan) [inline README.md]: Can you please keep the list of all models here? The general idea is to have something like "all models working" in the overview, but a list of models in each section, to make it clear which models are working.

**Comment by schwabe** [comment]
@Dunedan Did the first change. But could take me some time to get that lscpi output as that machine is primarely a OS X machine but I did boot Linux on it to run gparted. For the 2016 4C vs 2015 4C both are visible in the lspci, and they have different PCI ids according to the ioreg command on OS X (8086:1578 vs 8086,15d3)

**Comment by Dunedan** [comment]
> But could take me some time to get that lscpi output as that machine is primarely a OS X machine but I did boot Linux on it to run gparted.

No worries, I'll merge it as it is for now. As already mentioned it'd be awesome if you make another PR containing the output produced by `get-info.sh` once you happen to boot Linux again.

> For the 2016 4C vs 2015 4C both are visible in the lspci, and they have different PCI ids according to the ioreg command on OS X (8086:1578 vs 8086,15d3)

Indeed. That's the case for all the other models as well. I wonder why.

Thanks again for your contribution. :tada:

**Comment by Dunedan** [comment]
> No worries, I'll merge it as it is for now. As already mentioned it'd be awesome if you make another PR containing the output produced by get-info.sh once you happen to boot Linux again.

Never mind this comment. I was confusing you with somebody else with a MacBookPro14,3. Getting the more verbose lspci output would be still valuable though. :smile:

**Pull request merged**